### ISSUE TITLE
Bug for not contains

### ISSUE THREAD

**Issue by jurajsic** [VeriFIT/z3-noodler#157]
Current devele returns sat for
```
(set-logic QF_S)
(set-info :status unsat)
(declare-const a String)
(declare-const b String)
(assert (= a b))
(assert (str.contains "abc" a ))
(assert (not (str.contains "abcde" b )))
(check-sat)
(exit)
```
even though it should be unsat.

I think this will be fixed in #150, right?

**Comment by vhavlena** [comment]
Yes, I got `unsat` in the `ca-diseqs` branch.

**Comment by vhavlena** [comment]
I am closing the issue as it was solved by #150
